src/Compatibility/Microsoft.DotNet.PackageValidation/Validators/PackageValidatorOption.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.PackageValidation.Validators
+{
+    public struct PackageValidatorOption
```

Comment:
Same here, let's add trippe slash comments. For this one in particular, why making this a struct instead of a read-only struct? There are big performance differences when dealing with structs vs classes, so structs should usually be readonly when possible.